packages/next/next-server/lib/post-process.ts

```diff
@@ -243,6 +245,11 @@ function imageIsNotHidden(imgElement: HTMLElement): boolean {
   return true
 }
 
+// Currently only filters out svg images--could be made more specific in the future.
+function sourceIsSupportedType(imgSrc: string): boolean {
+  return !imgSrc.match(/\.svg/)
```

Comment:
```suggestion
  return !imgSrc.endsWith('.svg')
```

Reply:
Or this if it was intentional to leave out `$` at the end of the regex:

```suggestion
  return !imgSrc.includes('.svg')
```

Reply:
We'll use includes here because it could be using query params to transform the image, e.g. [imgix ](https://www.imgix.com/) or similar.